templates/zerver/confirm_continue_registration.html

```diff
@@ -23,28 +23,38 @@
                     <p>
                         {% trans %}
 
-                        You attempted to login using the email account
-                        {{ email }}, but this email address does not
-                        have a Zulip account in this organization. If
-                        you'd like, you can try to register a new
-                        Zulip account with this email address.
+                        No account found for {{ email }}. Would you like to register instead?
 
                         {% endtrans %}
                     </p>
                     {# TODO: Ideally, this should use whatever auth #}
```

Comment:
this comment should go after the "Go back to login" block (it refers to the "Continue to registration" block)